x/docs/md/intro/getting-started.md

```diff
@@ -3,7 +3,7 @@
 1. Download and configure the [command-line client](/cli).
 1. Fetch the first exercise (`exercism fetch TRACK_ID`).
 1. Write a solution to make the test suite pass.
-1. Submit your solution (`exercism submit path/to/file`).
+1. Submit your solution (`exercism submit path/to/file` or `exercism s path/to/file{.h,.cpp}`). 
```

Comment:
I think that the instruction is potentially useful for other tracks though.  Perhaps eventually there is an exercise that deals with IO with files, and the solution exists in a file that can be displayed along with the code submission, or something like that.  Perhaps there is a certain output that could be displayed and captured and submitted?  I don't know, bit I can imagine that this information is generally useful.

I have not tried to submit the test file (as an easy choice to test with) along with the exercise file to see how multiple file submissions work, though.